code/modules/mob/living/carbon/human/stripping.dm

```diff
@@ -6,6 +6,15 @@
 		user << browse(null, text("window=mob[src.name]"))
 		return FALSE
 
+	if(ishuman(user))
+		// If you're a human and don't have hands, you shouldn't be able to strip someone.
+		var/mob/living/carbon/human/H = user
+		var/obj/item/organ/external/r_hand = H.organs_by_name[BP_L_HAND]
+		var/obj/item/organ/external/l_hand = H.organs_by_name[BP_R_HAND]
```

Comment:
should use get_organ(BP_BLAH) here for both

Reply:
Code changes from earlier trying to make it work. Will do soon.